rust/datafusion/src/execution/physical_plan/planner.rs

```diff
@@ -318,12 +330,9 @@ impl DefaultPhysicalPlanner {
                         format!("{:#?}", input),
                     ));
                 }
-                let schema_ref = Arc::new((**schema).clone());
+                let schema_ref = Arc::new(schema.as_ref().clone());
```

Comment:
this `**` bothered me, so I switched to the different syntax. It has no intended behavior change

Reply:
I prefer to use idiomatic [language being written]. These changes are very welcoming from my side.